snippets/core/whats-new/whats-new-in-30/cs/http.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Net.Http;
+using System.Net.Security;
+using System.Net.Sockets;
+using System.Threading.Tasks;
+
+namespace whats_new
+{
+    public static class httptest
+    {
+        public static async Task Request()
+        {
+            // <SnippetRequest>
+            var client = new HttpClient() { BaseAddress = new Uri("https://localhost:5001") };
+
+            // HTTP/1.1 request
+            using (var response = await client.GetAsync("/"))
+                Console.WriteLine(response.Content);
+
+            // HTTP/2 request
+            using (var request = new HttpRequestMessage(HttpMethod.Get, "/") { Version = new Version(2, 0) })
+            using (var response = await client.SendAsync(request))
+                Console.WriteLine(response.Content);
```

Comment:
nit: I'd remove this indent so the `WriteLine` aligns with the `usings` above.

Reply:
Yeah I kept trying but Visual Studio 2019 wants it this way by default! So I figured I would leave it.